pkg/project/registry/project/rest.go

```diff
@@ -15,12 +16,12 @@ import (
 
 // REST implements the RESTStorage interface in terms of an Registry.
 type REST struct {
-	registry Registry
+	client kclient.NamespaceInterface
```

Comment:
Another reason this has to be this way is that we have to run on kube directly - we should make a note somewhere that it's illegal to reference a kube registry from an openshift storage object.